editor/src/clj/editor/validation.clj

```diff
@@ -87,7 +87,7 @@
                "'%s' must be between %d and %d"
                "'%s' must be between %f and %f")]
     (when (not (<= min v max))
-      (format tmpl name min max))))
+      (util/format* tmpl name min max))))
```

Comment:
Use `ROOT` locale when formatting numbers in error message. Stops floats from presenting as `123,456` with a Swedish locale, for example.